estate_account/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
commit message: start with a verb. and don't write what's 'obvious' in the odoo cosmos (bergide module are named with all the module it links. so typically, if I use your own words, it become
```
[ADD] estate_account: allows to create invoices when selling a property

then explain some more stuff if needed (not sure this is the case here)

task-xxxxx
```